docs/docs/contributor/local-setup/docker-setup.mdx

```diff
@@ -72,12 +72,12 @@ PG_DATABASE_URL=postgres://twenty:twenty@postgres:5432/default?connection_limit=
 
 ## Step 3: Build
 
-We provide an environment containerized with Docker and orchestrated with `docker-compose`.
+The project includes an environment containerized with Docker and orchestrated with `docker-compose`.
 This installation method will also provision a PostgreSQL container.
 
 :::info
 
-The configuration for the build is stored in the `infra/dev` folder, but you can run `make` commands directly from the root folder.
+The configuration for the build is present in the `infra/dev` folder, but you can run `make` commands directly from the root folder.
```

Comment:
I'm guessing the goal of removing stored was to make the sentence shorter not replace by a synonym :), we can just remove the word

Reply:
"is stored" was flagged as passive voice, but makes sense to remove it yeah